ncm-metaconfig/src/main/metaconfig/slurm/pan/schema.pan

```diff
@@ -86,7 +86,7 @@ type slurm_scheduler_parameters = {
     'bf_yield_interval' ? long(0..)
     'bf_yield_sleep' ? long(0..)
     'build_queue_timeout' ? long(0..)
-    'default_queue_depth' ? long(0..)
+    'default_5fqueue_5fdepth' ? long(0..)   # escaped version, otherwise unescaping produces some weird unicode
```

Comment:
I don't understand the need for this, are user really expected to assign to a path like this?
```pan
prefix "/software/components/metaconfig/services/{/etc/slurm/slurm.conf}/content"
"SchedulerParameters/default_5fqueue_5fdepth" = 128;
```
It seems very user-unfriendly, if there is a rendering issue, surely the schema is not the place to address it?

Reply:
Ah I see, so the issue is that escaping doesn't work in schema files?

Reply:
Ok, so I've just tested this and `{}` escaping seems to work fine in types, I would therefore prefer the following:
```suggestion
    '{default_queue_depth}' ? long(0..)   # escaped version, otherwise unescaping produces some weird unicode
```

Reply:
Wouldn't this change need to apply to every attribute containing _ ?

Reply:
I would have thought so, but perhaps the `_d` is what is tripping it up here.

Reply:
I still don't fully understand why the key needs to be escaped.